getx_demo/lib/screens/loan_application_screen.dart

```diff
@@ -0,0 +1,360 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+
+import 'package:get/get.dart';
+import 'package:http/http.dart' as http;
+
+import 'package:getx_demo/screens/transactions_screen.dart';
+import 'package:getx_demo/models/transaction.dart';
+import 'package:getx_demo/models/account.dart';
+import 'package:getx_demo/widgets/my_app_bar.dart';
+
+class LoanApplicationScreen extends StatelessWidget {
+  LoanApplicationScreen({
+    super.key,
+  });
+
+  static const route = '/loan-application-screen';
+
+  final salaryController = TextEditingController().obs;
+  final expensesController = TextEditingController().obs;
+  final loanAmountController = TextEditingController().obs;
+  final termController = TextEditingController().obs;
+
+  @override
+  Widget build(BuildContext context) {
+    final Account account = Get.find();
```

Comment:
`Account` is a model, it should not be held in `Get`. Current Account should be held in controller of the screen.